docs/semgrep-sc/sc-getting-started.md

```diff
@@ -21,135 +28,70 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Scanning open source dependencies
+# Scanning open source dependencies 
 
-<SscIntro />
+Semgrep Supply chain scans can be triggered from the Semgrep App interface.
 
-<AdmonitionSscLicense />
-
-![Semgrep Supply chain overview page](/img/sc-overview.png)
-_Figure 1_. Semgrep Supply Chain overview page.
+:::note
+Semgrep Supply Chain **supports monorepositories (monorepos)** by treating each subdirectory as its own repository. Findings are grouped under these repositories based on the lockfile or manifest file present in the subdirectory.
+:::
 
-Semgrep Supply Chain parses lockfiles for a list of dependencies, then scans your codebase using **rules **written with Semgrep's pattern syntax. Supply Chain rules specify the following:
+## Scanning with Semgrep Supply Chain through Semgrep App
 
-* The range of versions that contain the dependency's vulnerability.
-* A pattern for vulnerable code, such as passing in unsanitized data.
-* The severity of the vulnerability.
+:::tip
+This is the preferred method to enable and run Semgrep Supply Chain.
+:::
 
-Semgrep Supply Chain generates a **finding **when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
+Perform the following steps to create an account in Semgrep App and add (onboard) a repository for scanning. By adding a repository to Semgrep App, you are able to set up Semgrep Supply Chain scans and receive findings for vulnerabilities which you can triage and remediate.
 
-A finding is **unreachable** if the dependency contains a known vulnerability, but the vulnerable matching code is not used in your codebase.
+Read [Step 3](#step-3-enabling-semgrep-supply-chain-scans-from-within-semgrep-app) if you have already created an account in Semgrep App and added a repository.
```

Comment:
**Optional**: If these steps can be avoided, as they certainly can be, create a prerequisite section and link to particular procedures (follow the procedure in section X to Add, Install etc. you can even state how many steps there are in the procedure and link to it).
See this approach here (still WIP), see the blue prerequisite box:
<img width="999" alt="Screenshot 2022-12-07 at 16 13 35" src="https://user-images.githubusercontent.com/26669307/206216642-437f13c7-546b-44df-b526-328f3d7ccb66.png">
If you think that it is cleaner to include these procedures here, feel free to do so. But also, it is important to state to the user the steps that need to be followed in the linked document compared to following modularized items.

Reply:
@s-santillan this definitely makes sense. 😅 The initial comment was optional, thanks for pointing to really checking the Slack thread.